DEVELOPER.md

```diff
@@ -61,5 +61,25 @@ more about the tests.
 
 **_NOTE:_**  This command is currently Work in Progress. :construction::construction::construction::construction:
 
+
+### Debugging the Envoy Config
+An easy way to view the envoy config that Envoy Gateway is using is to expose the admin interface port (currently `19000`)
+by editing the envoy gateway service and adding the following port.
+
+```
+ports:
+- name: admin
+  port: 19000
+  protocol: TCP
+  targetPort: 19000
+```
+
+Next, use `kubectl port-forward` to  `kubectl port-forward svc/envoy-default-eg -n envoy-gateway-system 19000:19000`
```

Comment:
nit: you can skip the above text and directly port forward to the deployment with the same name